config/settings.yml.example

```diff
@@ -6,6 +6,9 @@
 #:ssl_certificate: ssl/certs/fqdn.pem
 #:ssl_ca_file: ssl/certs/ca.pem
 #:ssl_private_key: ssl/private_keys/fqdn.key
+# If you need to disable certain cipher suits use this option.
+# Only use this option if you know what you are doing!
```

Comment:
This isn't informative. If you think you need something along those lines, I would suggest a url to openssl ciphers...